run-tests-with-build-kogito-examples/pom.xml

```diff
@@ -61,42 +52,60 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
         <configuration>
+          <!-- kogito-runtimes tests were not designed to run in parallel -->
```

Comment:
I think also here we should refer `kogito-examples` rather than `kogito-runtimes`